static/css/slideshow-featured-events.css

```diff
@@ -195,18 +262,29 @@ div.image-holder {
   line-height:1.4rem;
   font-style:italic;
 }
-
+.collection-item > *{
+  font-size:1.4rem;
+  line-height:1.4rem;
+}
 /*featured events*/
 .events.featured {
   overflow-y: hidden;
-  height: 65vh !important;
+  height: 55vh !important;
 }
 .events.featured .eventItems > .collection{
-  height: 55vh;
+  height: 45vh;
   overflow-y:auto;
 }
 
-.events.featured img{
-  width:50%;
-  /*margin-left:33%;*/
+.events img{
+  height:35vh;
+  margin-left:3vh;
+  margin-right:3vh;
+  float:left;
+}
+
+/*//hide scroll bar*/
+::-webkit-scrollbar {
+    width: 0px;
+    background: transparent; /* make scrollbar transparent */
```

Comment:
I'd be curious about whether people like scrollbar or not (to show how many more events are upcoming)